kakao/src/main/kotlin/com/agoda/kakao/Builders.kt

```diff
@@ -44,10 +44,16 @@ class ViewBuilder {
     private val viewMatchers = arrayListOf<Matcher<View>>()
 
     /**
-     * Matches only view at given index, if there are multiple views that matches
+     * Matches only view at given [index], if there are multiple views that matches
+     *
+     * IMPORTANT: this matcher is single-use only, since it does not reset it's
+     * index counter due to specific espresso's matching process.
+     *
+     * If you need to match view with index multiple times, each time you should match
+     * with new instance of [withIndex]
```

Comment:
I think the better way is to provide the example as well? What do you think? @Alviere

Reply:
@VerachadW yeah, sure, I will add example here as well

Reply:
@VerachadW done